test/mox_test.exs

```diff
@@ -351,39 +351,84 @@ defmodule MoxTest do
 
       verify!()
       expect(CalcMock, :add, fn x, y -> x + y end)
+      expect(SciCalcOnlyMock, :exponent, fn x, y -> x * y end)
```

Comment:
**Annotation:** I realize that this implementation is actually multiplication (not an `exponent`) but these tests are focused on the `verify` method **not** the `expect` method, so I decided not to worry about this detail.